### PR TITLE
SettingsSchema converted to settings and script_contents removed

### PR DESCRIPTION
**Pull request by sarithapillai8**

### CODE REVIEW

File: src/model/ActivitySpec.ts [hunk 1]
```diff
@@ -18,11 +18,6 @@ export class ActivitySpec {
      */
     helpContents?: string
     
-    /**
-     * The WebView-compatible script that provides this Activity on mobile or desktop (IFrame) clients.
-     */
-    scriptContents?: string
```
Comment (avaidyam) [inline src/model/ActivitySpec.ts:24]: Please rename this to `executable` instead of deleting it.

File: src/model/ActivitySpec.ts [hunk 2]
```diff
@@ -36,7 +31,7 @@ export class ActivitySpec {
     /**
      * The Activity settings definition of an ActivitySpec.
      */
-    settingsSchema?: any
```
Comment (avaidyam) [inline src/model/ActivitySpec.ts:39]: Please rename `help_contents` and all the other properties too. 

> [...] `help_contents` to `description`, and all the `_schema` suffixes are dropped.

**Comment by sarithapillai8** [comment]
Please review

File: src/model/ActivitySpec.ts [hunk 3]
```diff
@@ -16,34 +16,30 @@ export class ActivitySpec {
...
      * The data definition of an ActivitySpec.
      */
-    dataSchema?: any
+    data?: any
     
     /**
      * The temporal slice data definition of an ActivitySpec.
      */
-    temporalSliceSchema?: any
+    temporalSlice?: any
```
Comment (avaidyam) [inline src/model/ActivitySpec.ts:34]: Please rename this to `temporalSlices`.

**Review by avaidyam — APPROVED** [review]

**Pull request merged**